index.html

```diff
@@ -8,21 +8,33 @@
 
   <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
   <link href="css/index.css" rel="stylesheet">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
 
- <!-- navigation code here -->
+<!-- navigation code here -->
+
```

Comment:
You probably also want to add a header around Spacewalkers and the nav. You can also have more than 1 header per html file. Some more info on that https://www.w3schools.com/tags/tag_header.asp